src/lib/builtins.pl

```diff
@@ -766,6 +766,10 @@
     read_term(Stream, Term, []).
     % read(Stream, Term).
 
+%% read(+Stream, -Term).
+%
+% Read Term from the stream Stream with default options. **NOTE** This is not a general predicate
```

Comment:
Regarding the phrasing "not a general predicate": Maybe this should instead clarify: "This predicate reads Prolog&nbsp;terms; for more general input, use for example `phrase_from_file/2` or `phrase_from_stream/2`."

Reply:
Please note that `read/2` is just here for convenience, it is a thin wrapper around `read_term/3` with default options. So for me IMHO this description would be enough: "Same as read_term/3 with all default options".